### PR TITLE
Rewrite Breadcrumb/LanguageSwitch with Providers and remove page placeholder

### PR DESCRIPTION
**Pull request by InnaBoitsun**
[AB#51378](https://dev.azure.com/orckestra001/da965986-c85b-43be-9915-219568b000e5/_workitems/edit/51378)

### CODE REVIEW

**Review by neexite — CHANGES_REQUESTED** [review]

File: src/Orckestra.Composer.Website/App_Data/PageTemplates/CategoryLandingPageLayout.cshtml [hunk 1]
```diff
@@ -12,34 +12,19 @@
...
-		  </f:param>
-		</f:function>
-	<div class="container-md">
-	    @Markup(Breadcrumb)
-	    @Markup(Body)
-	</div>
+    <body>
+        @this.Function("Composer.Header")
+        <div class="container-md">
+            @this.Function("Composer.Header.Breadcrumb")
```
[the remark this answers is not in the document]
Comment (InnaBoitsun) [reply src/Orckestra.Composer.Website/App_Data/PageTemplates/CategoryLandingPageLayout.cshtml]: fixed

File: src/Orckestra.Composer.Website/App_Data/PageTemplates/CategoryLandingPageLayout.cshtml [hunk 2]
```diff
@@ -12,34 +12,19 @@
...
-			  </body>
-			</html>
-		  </f:param>
-		</f:function>
-	<div class="container-md">
-	    @Markup(Breadcrumb)
-	    @Markup(Body)
-	</div>
+    <body>
+        @this.Function("Composer.Header")
```
[the remark this answers is not in the document]
Comment (InnaBoitsun) [reply src/Orckestra.Composer.Website/App_Data/PageTemplates/CategoryLandingPageLayout.cshtml]: Fixed

File: src/Orckestra.Composer.Website/App_Data/PageTemplates/AccountPageLayout.cshtml [hunk 3]
```diff
@@ -27,20 +21,11 @@
...
-                @Markup(LanguageSwitch)
-              </body>
-            </html>
-          </f:param>
-        </f:function>
-    <div class="container-md">
-        @Markup(Breadcrumb)
-        @Markup(Body)
-    </div>
+        @this.Function("Composer.Header")
```
[the remark this answers is not in the document]
Comment (InnaBoitsun) [reply src/Orckestra.Composer.Website/App_Data/PageTemplates/AccountPageLayout.cshtml]: Fixed

File: src/Orckestra.Composer.Website/App_Data/PageTemplates/CategoryPageLayout.cshtml [hunk 4]
```diff
@@ -12,34 +12,19 @@
...
-			  </body>
-			</html>
-		  </f:param>
-		</f:function>
-	<div class="container-md">
-	    @Markup(Breadcrumb)
-	    @Markup(Body)
-	</div>
+    <body>
+        @this.Function("Composer.Header")
```
[the remark this answers is not in the document]
Comment (InnaBoitsun) [reply src/Orckestra.Composer.Website/App_Data/PageTemplates/CategoryPageLayout.cshtml]: Fixed

**Review by neexite — APPROVED** [review]

**Pull request merged**